src/Get.tsx

```diff
@@ -96,28 +98,37 @@ export interface GetComponentState<T> {
  * debugging.
  */
 class ContextlessGet<T> extends React.Component<GetComponentProps<T>, Readonly<GetComponentState<T>>> {
-  private shouldFetchImmediately = () => !this.props.wait && !this.props.lazy;
-
-  readonly state: Readonly<GetComponentState<T>> = {
+  public readonly state: Readonly<GetComponentState<T>> = {
     data: null, // Means we don't _yet_ have data.
     response: null,
     error: "",
-    loading: this.shouldFetchImmediately(),
+    loading: false,
   };
 
-  componentDidMount() {
-    this.shouldFetchImmediately() && this.fetch();
+  public static getDerivedStateFromProps(props: Pick<GetComponentProps, "wait" | "lazy">) {
+    return { loading: !props.lazy };
```

Comment:
You don't use `wait` anymore? and what is `wait` btw? ^^

Reply:
I tried to be clear about what `wait` is [here](https://github.com/contiamo/restful-react/pull/11/commits/64b391ccac6bc1142b93e4aec70424dd8eaca1f6#diff-63d62a288bee27bda4e83b106d69c46dR66). 

I wonder how we can make it more clear. 🤔 

Basically, if `wait` is set, it doesn't render anything until there is data.